doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -184,6 +189,7 @@ label entries, so this is best used when bars also differ in style (e.g., by
 passing a list to *color*, as below.)
 
 .. plot::
+    :alt: bar chart: blue bar height 10, orange bar height 20, green bar height 15 legend with blue box labeled a, orange box labeled b, and green box labeled c
```

Comment:
```suggestion
    :alt: bar chart: blue bar height 10, orange bar height 20, green bar height 15; legend with blue box labeled a, orange box labeled b, and green box labeled c.
```

Reply:
Should the num

Reply:
I'm not sure that we have a specific style guide that specifies either way. I just know that the first word in the sentence is better spelled out. IIRC, for in-sentence numbers it's usually only for 'small' numbers, so these are probably too large.